src/gpgme.cc

```diff
@@ -167,11 +161,7 @@ static inline void init_lib() {
 }
 
 static inline void rewind(Data * d) {
```

Comment:
Would you mind dropping `rewind()` entirely and replacing all calls to it with `d->rewind()`? The API differences between GPGME versions were the reasons for its existence.

Reply:
Excellent suggestion, thanks!